process-document/src/test/kotlin/com/ritense/processdocument/service/CorrelationServiceIntTest.kt

```diff
@@ -0,0 +1,219 @@
+package com.ritense.processdocument.service
+
+import com.fasterxml.jackson.core.JsonProcessingException
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.ritense.document.domain.Document
+import com.ritense.document.domain.impl.JsonSchemaDocumentId
+import com.ritense.document.domain.impl.request.NewDocumentRequest
+import com.ritense.document.service.DocumentService
+import com.ritense.processdocument.BaseIntegrationTest
+import com.ritense.processdocument.repository.ProcessDocumentInstanceRepository
+import com.ritense.valtimo.service.CamundaProcessService
+import org.camunda.bpm.engine.RuntimeService
+import org.camunda.bpm.engine.TaskService
+import org.camunda.bpm.engine.task.Task
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.transaction.annotation.Transactional
+import java.util.function.Consumer
+import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
+import kotlin.test.assertNull
+
+@Transactional
+class CorrelationServiceIntTest: BaseIntegrationTest() {
```

Comment:
Tests in this file are not following the naming convention we use for other tests.